src/SortFun.ts

```diff
@@ -0,0 +1,66 @@
+import { actualData, tableCreate } from "./script.ts";
+import { FilterArr } from "./filterAndSearchFun.ts";
+const tableBody = document.querySelector(".table-body")! as HTMLTableElement;
+
+let dirFlag = 1;
+const sortButton = document.querySelector(".sort-button")! as HTMLImageElement;
+// sort functionality
+//////////////////////////////////////
+const sortFun = () => {
+  console.log(FilterArr);
+  let arrayToSort = actualData.employee;
+  if (FilterArr.length !== 0) arrayToSort = FilterArr;
+
+  let arrToRender = arrayToSort.sort(
+    (
+      a: {
```

Comment:
extract custom types to a separate file and import the type here. Don't redefine types repeatedly